pkg/lib/localstore/ftsearch/ftsearch.go

```diff
@@ -174,8 +175,17 @@ func (f *ftSearch) getTerms(qry string) []string {
 	return terms
 }
 
-func (f *ftSearch) doSearch(queries []query.Query) (results []string, err error) {
-	searchRequest := bleve.NewSearchRequest(bleve.NewDisjunctionQuery(queries...))
+func (f *ftSearch) doSearch(spaceID string, queries []query.Query) (results []string, err error) {
+	var rootQuery query.Query
+	disjunctionQuery := bleve.NewDisjunctionQuery(queries...)
+	if spaceID != "" {
+		spaceQuery := bleve.NewMatchQuery(spaceID)
+		rootQuery = bleve.NewConjunctionQuery(disjunctionQuery, spaceQuery)
+	} else {
+		rootQuery = disjunctionQuery
+	}
+
+	searchRequest := bleve.NewSearchRequest(rootQuery)
 	searchRequest.Size = 100
 	searchRequest.Explain = true
```

Comment:
tests for fts please

Reply:
Will be discussed and implemented under https://linear.app/anytype/issue/GO-1734/graylog-should-have-ability-to-write-the-file

Reply:
Added unit tests for multi space